ledger/metrics/src/main/scala/com/daml/metrics/Metrics.scala

```diff
@@ -402,6 +404,11 @@ class Metrics(val registry: MetricRegistry) {
       def currentRecordTimeLag(value: () => Long): Gauge[Nothing] =
         gauge(prefix :+ "current_record_time_lag", () => () => value())
 
+      // Note: this counts individual updates
+      val receivedUpdates: Counter = registry.counter(prefix :+ "received_updates")
+
+      // Note: these measure update batches
+      val updateBatchSizes: Histogram = registry.histogram(prefix :+ "update_batch_sizes")
```

Comment:
If you think the names are lacking I would suggest finding a way of including the lacking information in the same itself, rather than leaving it into the comment.